tester/orb.yaml

```diff
@@ -157,6 +157,11 @@ jobs:
         description: >
           Working directory used when running docker-compose commands.
         type: string
+      compose_files:
```

Comment:
Let's make this more generic (`compose_args`) since we have folks passing the full arguments rather than just a list of files. If we really want to trim it down to only be `files`, we should avoid having users pass the `-f` flag... but at that point, we'd just be hiding functionality for no benefit so /shrug

Reply:
yep, makes sense. Check now <3